### ISSUE TITLE
Why isn't YearMonth.PlusMonths() available?

### ISSUE THREAD

**Issue by nhirschey** [nodatime/nodatime#1627]
I cannot access YearMonth.PlusMonths() and it's not showing up on the API documentation either: https://nodatime.org/3.0.x/api/NodaTime.YearMonth.html#methods

It should have been added in this pull request: https://github.com/nodatime/nodatime/pull/1550

```fsharp
open NodaTime
open NodaTime.Calendars

let ym = YearMonth(Era.Common,1919,1)
ym.PlusMonths(1) // does not exist.
```

**Comment by jskeet** [comment]
It was added, but we haven't done a new minor release since then. It hasn't seemed worth it for just the tiny number of features so far. As there don't seem to be any other features coming imminently, I could potentially do another release at some point - but it won't be for a little while, as I'm very busy with other things and it's a non-trivial amount of work.

**Comment by nhirschey** [comment]
Thank you for the clarity. Completely understood. 

I raised this issue only because I thought it might be some sort of bug because I had seen the 3.0.5 release pushed to Nuget after those pull requests were merged. But I clearly wasn't tracking the correct branches ;)

I will extend the class myself for now until the official version is released. Thank you for your work and adding this YearMonth class (it is very useful for time-series work with monthly data).
